### PR TITLE
[W5.7][T10-1]Bryan Wong Wei Heng

### PR DESCRIPTION
**Pull request by bryanwongweiheng**
change find command to case-insensitive

### CODE REVIEW

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -79,14 +79,17 @@ Format: `find KEYWORD [MORE_KEYWORDS]`
 
 [NOTE]
 ====
-The search is case sensitive, the order of the keywords does not matter, only the name is searched,
+The search is not case sensitive, the order of the keywords does not matter, only the name is searched,
```
Comment (pangjiahao) [inline docs/UserGuide.adoc:82]: I think this is a good idea, makes it easier to search for people without bothering with upper or lower case

**Review by pangjiahao — COMMENTED** [review]
Looks great.

File: docs/UserGuide.adoc (hunk 1 above)
Comment (yunpengn) [reply docs/UserGuide.adoc:82]: Hi, Jia Hao.

Maybe you want to use inline comments for more specific things. For overall feedback, leave it in the review summary.

**Review by yunpengn — APPROVED** [review]
Good idea. Please close this PR after viewing my comments.

**Pull request closed without merging**